overlord/snapstate/autorefresh_test.go

```diff
@@ -1172,7 +1172,7 @@ func (s *autoRefreshTestSuite) TestTooSoonError(c *C) {
 	c.Check(snapstate.TooSoonError{}.Error(), Equals, "cannot auto-refresh so soon")
 }
 
-func setStoreAccess(s *state.State, access string) {
+func setStoreAccess(s *state.State, access any) {
```

Comment:
historically we never user "any" afaict, we tend to spell out interface{}, the std lib seems to do that mostly too

Reply:
Good to know, fixed!